opendevin/controller/agent_controller.py

```diff
@@ -188,6 +195,7 @@ async def on_event(self, event: Event):
                 logger.info(event, extra={'msg_type': 'OBSERVATION'})
 
     def reset_task(self):
+        self.almost_stuck = 0
```

Comment:
This is currently computed, but it's not used, it doesn't add or change anything in the prompt. We can make and evaluate changes based on it in follow-up PRs.